test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -611,6 +612,26 @@ foo
             }
         }
     }
+
+    Context "CustomPipeName startup tests" {
+
+        It "Should create pipe file if CustomPipeName is specified" {
+            $pipeName = [System.IO.Path]::GetRandomFileName()
+            $pipePath = Get-PipePath $pipeName
+
+            # The pipePath should be created by the time the -Command is executed.
+            pwsh -CustomPipeName $pipeName -Command "Test-Path '$pipePath'" | Should -BeTrue
```

Comment:
use `$powershell` instead. Using `pwsh` will call the installed `pwsh` on the system, not the one which we build.

Reply:
If what you say is true, then none of my CustomPipeName tests would be passing, right? I'm willing to change it though. Is that a magic variable or something?

Reply:
Probably set during the build or on test initialization with Start-PSPester, I'd imagine. 🙂

Reply:
Oh I see what you're saying now :)

Reply:
I've fixed this so it's consistent